financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -45,19 +48,35 @@ internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity(), Ma
     @Inject
     lateinit var navigationManager: NavigationManager
 
+    private lateinit var imageLoader: StripeImageLoader
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        imageLoader = StripeImageLoader(context = this)
         viewModel.activityRetainedComponent.inject(this)
         viewModel.onEach { postInvalidate() }
         setContent {
             FinancialConnectionsTheme {
-                Column {
-                    Box(modifier = Modifier.weight(1f)) { NavHost() }
+                ActivityCompositionLocalProvider {
+                    Column {
+                        Box(modifier = Modifier.weight(1f)) { NavHost() }
+                    }
                 }
             }
         }
     }
 
+    @Composable
+    internal fun ActivityCompositionLocalProvider(
```

Comment:
Should this be in a standalone extension function with some documentation as they are required to use `StripeImage`?

Reply:
Actually you make a good point here. `StripeImage` definitely depends on `StripeImageLoader`, but not on how the dependency is created / provided.

We should be flexible on how each SDK manages its `StripeImageLoader` dependency, as it is stateful and scopes vary across SDKs. I reversed the imageLoader dependency so that it has to be passed as a parameter.

`ActivityCompositionLocalProvider` is how FinancialConnections decides to instantiate and scope ImageLoader, since we're on a single activity and we want the loader to be scoped to it.

Reply:
https://github.com/stripe/stripe-android/pull/5307/commits/8d4670dab7c324c046f0e3a074bb6b4e5a77bf86